confd/pkg/resource/template/template_funcs.go

```diff
@@ -31,6 +35,7 @@ func newFuncMap() map[string]interface{} {
 	m["datetime"] = time.Now
 	m["toUpper"] = strings.ToUpper
 	m["toLower"] = strings.ToLower
+	m["title"] = title.String
```

Comment:
I don't think `title` is used in the templates, so let's revert this.  (Guessing you used it during development but then changed that?)